### DISCUSSION TITLE
UI configuration now showing any options.

### DISCUSSION THREAD

**Discussion by RyanGibb** [basnijholt/adaptive-lighting#1053] [Q&A]
I'm wondering if this is a bug, but I'm posting here first in case it's user error.

When I try and configure the component through the UI I'm met with:
![image](https://github.com/user-attachments/assets/013762bf-d60a-4d09-b20b-c07cc5f58d8b)

Note the lack of options.

My configuration is:
```
adaptive_lighting:
  lights:
  - light.bed_left
  - light.bed_right
  - light.ceil
  - light.strip
```

I'm on version 1.22.0 of adaptive_lighting.

**Comment by FezVrasta** [comment]
I have the same issue but I only added `adaptive_lighting:` to the YAML like explained in the docs... What am I supposed to do to make the UI show the options?
